src/app/app/app-providers/fetch-config-providers.ts

```diff
@@ -69,6 +70,10 @@ export const getProviders = (conf): any[] => {
 
 export const fetchConfigProviders = fetch('/assets/config/app.config.json')
 	.then(response => response.json())
+	.then(conf => {
+		AnsynLogger.activeLogger = new AnsynLogger(conf.loggerConfig.env);
```

Comment:
why different environment for the logger? we already have an environment config

Reply:
This is according to design.

Reply:
everything is static now